### PR TITLE
Date column reading and writing with zone offset fixed

### PR DESCRIPTION
**Pull request by neverlee**
Sorry, I opened the older pr without a complete testing. It did not solve this problem [issue140](https://github.com/kshvakov/clickhouse/issues/140)

Now, I open this pr for this problem.

Insert by golang:
image

Query by clickhouse-client:
image

Query by golang:
image

### CODE REVIEW

File: lib/column/date.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		return nil, err
+	}
+	return time.Unix(int64(sec)*24*3600-dt.offset, 0).In(dt.Timezone), nil
+}
+
+func (dt *Date) Write(encoder *binary.Encoder, v interface{}) error {
+	var timestamp int64
+	switch value := v.(type) {
+	case time.Time:
+		_, offset := value.Zone()
```
[the remark this answers is not in the document]
Comment (neverlee) [reply lib/column/date.go]: The "time.Time" has it's own "time.Location", so "value.Zone()",
The int timestamp has no  "time.Location", so using the default column timezone offset. (Do you think this offset is necessary?)

**Comment by kshvakov** [comment]
Can you add some tests for this?

**Comment by neverlee** [comment]
Yeah, no problem. I will add some tests today.

**Comment by neverlee** [comment]
The tests of data column have done.

**Comment by kshvakov** [comment]
Thank you!

**Pull request merged**